build_tools/azure/test_docs.sh

```diff
@@ -4,8 +4,10 @@ set -e
 
 if [[ "$DISTRIB" =~ ^conda.* ]]; then
     source activate $VIRTUALENV
+    conda install scikit-image -y
 elif [[ "$DISTRIB" == "ubuntu" ]]; then
     source $VIRTUALENV/bin/activate
+    pip install scikit-image
```

Comment:
you probably want to install scikit-image from the system

Reply:
IIRC our min supported scikit-image is newer than the one available in ubuntu. And in `install.sh` we do install a bunch of libs in the environment, so I don't see why this should fail.